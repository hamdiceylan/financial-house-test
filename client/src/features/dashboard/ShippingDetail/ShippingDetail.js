import React from 'react'
import { Grid, Header } from 'semantic-ui-react'

function ShippingDetail({client}) {
  return (
    <Grid.Column width={8}>
        <Header icon='shipping' content='Shipping Details'/>
        <p>Address: <strong>{client.shippingAddress1} {client.shippingAddress2}</strong></p>
        <p>Shipping City <strong>{client.shippingCity}</strong></p>
        <p>Shipping Country: <strong>{client.shippingCountry}</strong></p>
        <p>Shipping Post Code: <strong>{client.shippingPostcode}</strong></p>
    </Grid.Column>
  )
}

export default ShippingDetail
