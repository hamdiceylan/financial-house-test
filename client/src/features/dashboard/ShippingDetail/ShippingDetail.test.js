import React from 'react'
import { shallow } from 'enzyme'
import ShippingDetail from './ShippingDetail';

describe('ShippingDetail', () =>{
    const props = {
        client : {
            shippingAddress1: 'address 1',
            shippingAddress2: 'address 2',
            shippingCity: 'city',
            shippingCountry: 'country',
            shippingPostcode: 'post code'
        }
    }
    const shippingDetail = shallow(<ShippingDetail {...props} />);

    it('renders correctly', () => {
        expect(shippingDetail).toMatchSnapshot();
    })

})

