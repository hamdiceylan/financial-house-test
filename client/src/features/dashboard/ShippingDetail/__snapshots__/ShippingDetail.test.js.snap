// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShippingDetail renders correctly 1`] = `ShallowWrapper {}`;
